crates/typst/src/visualize/path.rs

```diff
@@ -147,14 +148,20 @@ fn layout_path(
 
     // Prepare fill and stroke.
     let fill = elem.fill(styles);
+    let fill_rule = elem.fill_rule.unwrap_or_default();
```

Comment:
```suggestion
    let fill_rule = elem.fill_rule(styles);
```
otherwise, set rules won't work